my.new.project/AmpegManualTestProject.txt

```diff
@@ -0,0 +1,80 @@
+Title: Carousel-test-01
```

Comment:
Use plain English titles -
"Carousel cycles through images"

The ID should be a separate field